components/upload/chunk-upload.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Chunk Upload
+page_title: Chunk Upload
+description: Enable and configure chunk upload in Upload for Blazor.
+slug: upload-chunk
+tags: telerik,blazor,upload,chunk
+published: True
+position: 3
+---
+
+# Chunk Upload
+
+Chunk upload enables file uploads by dividing the file into smaller parts (chunks) and sending them in multiple requests to the remote endpoint, where they are reassembled into the final file.
+
+## Basics
+
+To setup the feature, use the `UploadChunkSettings` tag, which is nested inside `UploadSettings`. The tag includes the following parameters:
+
+| Parameter | Type and Default Value | Description |
+|----------|----------|----------|
+| `AutoRetryAfter` | `double` <br/> (100) | Specifies the amount of time in milliseconds after which a failed chunk upload request will be retried.  
+| `Enabled` | `bool` <br/> (`true`) | Specifies if the chunk upload is enabled.  
+| `MaxAutoRetries` | `int` <br/> (1) | Specifies the number of attempts to retry uploading a failed chunk.  
+| `MetadataField` | `string` <br/> (`chunkMetadata`) | Specifies the name of the variable that will receive the chunk upload metadata in the remote endpoint.  
+| `Resumable` | `bool` <br/> (`true`) | Specifies if the file upload process could be paused and later resumed.  
+| `Size` | `double` <br/> (1024 * 1024) | The size of the chunks in bytes.
+
+## Events
+
+The Upload exposes several relevant events. You can find related examples in the [Events](slug:upload-events) article.
+
+* `OnPause` - fires when the user clicks on the pause button during chunk upload.
+* `PageSizeChanged` - fires when the user clicks on the "resume" button during chunk upload.
+
+>caption Enable Chunk Upload
+
+````RAZOR
```

Comment:
I suggest skipping the REPL here as we do not have a controller and the users won't be able to actually upload files.